<?xml version="1.0"?>
<launch>
  <arg name="candidates_topic" default="/targets/candidates" />
  <arg name="selected_topic" default="/targets/selected_pos" />
  <arg name="map_frame" default="map" />
  <arg name="base_frame" default="base_link" />
  <arg name="min_confidence" default="0.05" />

  <node pkg="target_follower" type="target_candidate_selector.py" name="target_candidate_selector" output="screen">
    <param name="candidates_topic" value="$(arg candidates_topic)" />
    <param name="selected_topic" value="$(arg selected_topic)" />
    <param name="map_frame" value="$(arg map_frame)" />
    <param name="base_frame" value="$(arg base_frame)" />
    <param name="min_confidence" value="$(arg min_confidence)" />
  </node>
</launch>
